### PR TITLE
Web/refactor/convert mui components to styled

### PR DESCRIPTION
**Pull request by jkurlowicz**
Change styling from JSS to Emotion so we can upgrade MUI.

part of #5299 

**Web frontend checklist**
- [x] Formatted my code with `yarn format`
- [x] There are no warnings from `yarn lint --fix`
- [x] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [x] All tests pass
- [x] Clicked around my changes running locally and it works
- [x] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/web/components/Alert.tsx [hunk 1]
```diff
@@ -20,20 +21,18 @@ export default function Alert({
...
-  const classes = useStyles();
-
   const oldErrorKey = Object.keys(grpcErrorStrings).find(
     (oldError): oldError is ObscureGrpcErrorMessages =>
       children.includes(oldError)
   );
 
   return (
-    <MuiAlert {...otherProps} className={classNames(classes.root, className)}>
+    <StyledAlert {...otherProps} className={classNames(className)}>
```
Comment (nabramow) [inline app/web/components/Alert.tsx]: I think you can get rid of the `classNames` function and import and just put the `className` prop here now.

**Comment by nabramow** [comment]
Okay tested the branch and seems to be working well. Also on mobile! Just that one small change then I'll approve!

File: app/web/components/Alert.tsx (hunk 1 above)
Comment (jkurlowicz) [reply app/web/components/Alert.tsx]: done :)

**Review by nabramow — APPROVED** [review]
Looks good to me! 🚀

**Pull request merged**